openslides/agenda/static/js/agenda/csv.js

```diff
@@ -5,16 +5,20 @@
 angular.module('OpenSlidesApp.agenda.csv', [])
 
 .factory('AgendaCsvExport', [
-    function () {
+    'HumanTimeConverter',
+    function (HumanTimeConverter) {
         return function (element, agenda) {
             var csvRows = [
                 ['title', 'text', 'duration', 'comment', 'is_hidden'],
             ];
             _.forEach(agenda, function (item) {
                 var row = [];
+                var duration = item.duration ? HumanTimeConverter.secondsToHumanTime(item.duration*60,
+                        { seconds: 'disabled',
+                            hours: 'enabled' }) : '';
                 row.push('"' + (item.title || '') + '"');
                 row.push('"' + (item.text || '') + '"');
-                row.push('"' + (item.duration || '') + '"');
+                row.push('"' + (duration || '') + '"');
```

Comment:
duration is already an empty string (if item.duration was falsy) so we do not have to use the or clause again.